pkgs/os-specific/linux/systemd/default.nix

```
@@ -226,6 +230,19 @@ stdenv.mkDerivation (finalAttrs: {
     ./0015-tpm2_context_init-fix-driver-name-checking.patch
     ./0016-systemctl-edit-suggest-systemdctl-edit-runtime-on-sy.patch
     ./0017-meson.build-do-not-create-systemdstatedir.patch
+
+    # https://github.com/systemd/systemd/pull/33258
+    # Remove after 256.3
```

I don't see a backport PR.

```suggestion
    # Remove after these changes are included in a systemd release
```

---

IIUC, they don't do backports the way we do with nixpkgs. The PR does have the `needs-stable-backport` label, which means it *should* just end up pushed to the 256 branch before the next release (although it didn't before 256.2, but those were only days apart)